bench/tests/test_base.py

```diff
@@ -15,7 +15,7 @@
 if sys.version_info.major == 2:
 	FRAPPE_BRANCH = "version-12"
 else:
-	FRAPPE_BRANCH = "develop"
+	FRAPPE_BRANCH = "version-13"
```

Comment:
This removes any way of knowing if bench is compatible with the latest Frappe develop. We might have to add more tests to ensure that.

What is the reason for doing this?

Reply:
The idea was the latest stable version should be tested on priority over develop as that's what would be used most

If we want to write more tests for each version we can, but the tests will need to be updated for newer versions develop aside.

What do you suggest we do? Should I just keep it develop?

Reply:
> The idea was the latest stable version should be tested on priority over develop as that's what would be used most


For testing version compatibility of Frappe with the Bench CLI, I think we still do test the relevant bits (bench creation, requirements resolution, assets building) in the Frappe CI. Anyway, Frappe and ERPNext CIs test this on push and branch merges daily, as compared to the relatively low frequency tests being run here.


> If we want to write more tests for each version we can, but the tests will need to be updated for newer versions develop aside.


Not sure if testing for each major version separately is the way to go. Frappe and Bench are fairly de-coupled in the cases these test suites are written for.


> What do you suggest we do? Should I just keep it develop? @gavindsouza


I'd stick with leaving this as `develop`. Don't have clear enough reasons to make any change.

Reply:
Okay, makes sense